src/app/shared/components/main-nav/main-nav.component.spec.ts

```diff
@@ -18,7 +18,7 @@ function mockStore() {
   return jasmine.createSpyObj('Store', ['select']);
 }
 
-describe('MainNavComponent', () => {
+xdescribe('MainNavComponent', () => {
```

Comment:
is it okay?

Reply:
it has only one test and the test is commented. But I completelly agree about testing.